Error setting up environment with docker

<!--
Instructions:
 * Fill out the sections below, replace …'s with information about your issue
 * Use the 'preview' function above this text box to verify formatting before submitting
-->

## Observed behavior
When building the base docker image and installing python base requirements, some python dependency throws an error pointing out that the minimum version of python required is 3.5.

## Errors and logs

```
Downloading https://files.pythonhosted.org/packages/b7/81/c04fb9be62657d4dce8aa2d99fde258a3af1cd77ec72af525593e9560127/ruamel.yaml.clib-0.2.4.tar.gz (180kB)
    Complete output from command python setup.py egg_info:
    minimum python version(s): [(3, 5)]
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-CyxhF5/ruamel.yaml.clib/
The command '/bin/sh -c cd /kolibri     && pip install -r requirements/dev.txt     && pip install -r requirements/build.txt     && pip install -r requirements/test.txt     && yarn install --network-timeout 100000' returned a non-zero code: 1
make: *** [Makefile:260: docker-build-base] Error 1
```


## Expected behavior
Docker installs the system without errors in an simple way.

## Steps to reproduce
Run `make docker-build-base` as indicated in https://kolibri-dev.readthedocs.io/en/latest/getting_started.html#id2


## Context
* kolibri version: latest
* Operating system: Ubuntu 20.04 LTS
* Kernel: Linux 5.11.0-27-generic

HI @rtibbles. Sure, it occurs on the step 9 of the base image docker build:
```
Step 8/9 : COPY . /kolibri
 ---> 56d1f7631997
Step 9/9 : RUN cd /kolibri     && pip install -r requirements/dev.txt     && pip install -r requirements/build.txt     && pip install -r requirements/test.txt     && yarn install --network-timeout 100000
 ---> Running in 1deb8412094e
Collecting diskcache==4.1.0 (from -r requirements/base.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/ee/cc/d992e1d886d5ce15d2622c2e89b6de52b48312c6f05e34b7ee881b4ccb02/diskcache-4.1.0-py2.py3-none-any.whl (44kB)

... Collect some dependencies more ...

Collecting pyparsing>=2.0.2 (from packaging>=17.1->tox==3.1.3->-r requirements/dev.txt (line 5))
  Downloading https://files.pythonhosted.org/packages/8a/bb/488841f56197b13700afd5658fc279a2025a39e22449b7cf29864669b15d/pyparsing-2.4.7-py2.py3-none-any.whl (67kB)
Requirement already satisfied: jinja2 in /usr/lib/python2.7/dist-packages (from coreschema>=0.0.4->drf-yasg==1.17.1->-r requirements/dev.txt (line 8))
Collecting ruamel.yaml.clib>=0.1.2; platform_python_implementation == "CPython" and python_version < "3.10" (from ruamel.yaml>=0.15.34->drf-yasg==1.17.1->-r requirements/dev.txt (line 8))
  Downloading https://files.pythonhosted.org/packages/b7/81/c04fb9be62657d4dce8aa2d99fde258a3af1cd77ec72af525593e9560127/ruamel.yaml.clib-0.2.4.tar.gz (180kB)
    Complete output from command python setup.py egg_info:
    minimum python version(s): [(3, 5)]
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-Z1reFq/ruamel.yaml.clib/
The command '/bin/sh -c cd /kolibri     && pip install -r requirements/dev.txt     && pip install -r requirements/build.txt     && pip install -r requirements/test.txt     && yarn install --network-timeout 100000' returned a non-zero code: 1
make: *** [Makefile:260: docker-build-base] Error 1
```

--- a/requirements/dev.txt
+++ b/requirements/dev.txt
@@ -5,6 +5,7 @@ pre-commit==1.15.1
 tox==3.1.3
 django-debug-panel==0.8.3
 django-debug-toolbar==1.9.1
+ruamel.yaml.clib==0.2.2
 drf-yasg==1.17.1
 coreapi==2.3.3
 nodeenv==1.3.3

